API.yaml

```diff
@@ -1246,6 +1246,8 @@ paths:
           One of 'requested_table_index' or 'requested_times' should be present. If 'requested_table_index' is used it is the starting index of the lines to be returned. If 'requested_times' is used it should contain an array with a single timestamp and the returned lines starting at the given timestamp (or the nearest following) will be returned.
           The 'requested_table_count' is the number of lines that should be returned.
           When 'requested_table_column_ids' is absent all columns are returned. When present it is the array of requested columnIds.
+          Use 'table_search_expressions' for search providing a map of col regular expresions. Returned lines that match the search expression will be tagged.
+          Use 'table_search_direction' to specifiy search direction [NEXT, PREVIOUS]. If present, 'requested_table_count' tagged events starting from the first matching event are returned. If no match, an empty list of events is returned.
```

Comment:
I'm not sure, when you say 'tagged events' does it mean only matching events are returned? I don't think that is the case?

Reply:
I'll update it.